process_request/helpers.py

```diff
@@ -3,6 +3,164 @@
 from rapidfuzz import fuzz
 
 
+def get_container_indicators(instance):
+    """Takes ArchivesSpace JSON instance data with resolved top containers and returns a container indicator.
+
+    Args:
+        instance (dict): ArchivesSpace instance information that has resolved top containers and digital objects.
+
+    Returns:
+        string: A concatenated string containing the container type and container indicator, or digital object title.
+    """
+    if instance.get("instance_type") == "digital_object":
+        return "Digital Object: " + instance.get("digital_object").get("_resolved").get("title")
```

Comment:
```suggestion
        return "Digital Object:  {}".format(instance.get("digital_object").get("_resolved").get("title"))
```
Format strings are easier to read!